src/integer/intLib.sml

```diff
@@ -64,6 +64,84 @@ val ARITH_CONV = Omega.OMEGA_CONV
 val ARITH_TAC = Omega.OMEGA_TAC
 val ARITH_PROVE = Omega.OMEGA_PROVE
 
+(* ------------------------------------------------------------------------- *)
+(* Instantiate the normalizer (code below are ported from HOL-Light)         *)
+(* ------------------------------------------------------------------------- *)
+
+local
+  val sth = prove
```

Comment:
was this code moved from somewhere else? I believe this was significantly refactored, can you make sure it is the new version that is moved?

Reply:
Your refactoring was in the just merged PR which I cannot depend on. It's moving theorems into the old `integerRingScript.sml`, but now this file has to move to examples, and the updated `intLib` need to provide `INT_RING_TAC` which depends on `INT_RING` which depends on `INT_POLY_CONV`, which depends on a `sth` theorem aka your `INT_POLY_CONV_sth` moved to examples. Since `intLib` cannot depend on code in `examples`, I found it's reasonable to do this entire work assuming your previous PR doesn't exist.

If you want, you can do this refactoring again. I don't know where I should put `INT_POLY_CONV_sth` into.